docs/meshstack.building-aws-quickstart-guide.md

```diff
@@ -0,0 +1,182 @@
+      git tag, branch, or commit.
+    - If you don’t use a self-hosted git repository, a “Test Connection” button will appear.
+    - Click this button and if everything is correct you should see all Terraform files related to your AWS S3 bucket.
+        - If anything has gone wrong, you will receive an error message instead. Analyze the given error, try to
+          resolve the issue, and try again.
+    - For “Execution Mode” leave this as it is. In complex scenarios, you can take over the status reporting of your
+      Building Block using meshStack API’s.
+    - For “Deletion Mode” leave this as it is too. We highly recommend using “Delete Resources” which means that
+      meshStack will clean up the underlying infrastructure resources upon deletion using `terraform destroy`.
+7. **Entering Inputs**
+
+   If you haven’t already, click “Next” on the implementation page.
+
+   You will land on the “Dependencies” page.
+
+    - meshStack can work with [Dependencies between Building Blocks]. We will skip this in this quickstart guide.
+
+   Click “Next” again and you should see a modal pop-up with imported inputs, authentication, and outputs.
+
+   meshStack read out the Terraform files and automatically generated fitting inputs based on the defined variables, outputs, and providers.
+
+   Leave all of them checked and click “Add selected”.
+
+   We will now see all variables that are defined in Terraform plus the AWS authentication environment variables that
+   are needed to manage resources in AWS. Let’s fill those in:
+
+   (If you are unsure what rights are needed, take a look at our
+   [Terraform IAM definition](https://github.com/meshcloud/collie-hub/blob/main/kit/aws/buildingblocks/s3_bucket/backplane/iam.tf)
+   for this quickstart guide)
+
+    - For `AWS_ACCESS_KEY_ID` add the ID of an Access Key in AWS that has the right to manage S3 resources in the “Static Value” field.
+    - For `AWS_SECRET_ACCESS_KEY` enter the secret of the Access Key in AWS in the “Static Value” field.
+8. **Entering Outputs**
+
+   Move on to the Outputs page by clicking “Next” on the bottom right.
+
+   The outputs will also be there already based on the import from before. You can leave all of these as they are and click “Create Definition”.
+
+🎉Your definition is created, well done! The hard work is finished. Let’s get to testing your new definition and make sure it works as expected.
+
+## Testing the Definition
+
+After the previous step, you should have landed on the control plane of your new building block definition.
+There should be a wizard explaining what the next step is. Let’s follow those recommendations.
+
+1. **Deploy & Start Testing**
+
+   Under “Test definition” you’ll see a blue button “Deploy & Start Testing”. Click on it.
+
+2. **Enter Inputs**
+
+   You will be prompted to enter all Inputs that were configured before. Enter these with some sensible values and hit “Deploy Building Block”.
```

Comment:
I think it makes sense to provide examples for if it makes sense the information to be put in